### ISSUE TITLE
Any plans to update to OpenDRIVE 1.8?

### ISSUE THREAD

**Issue by johschmitz** [esmini/esmini#592]
I am trying to implement stop line signals but the length argument is missing before OpenDRIVE 1.8 (see https://publications.pages.asam.net/standards/ASAM_OpenDRIVE/ASAM_OpenDRIVE_Specification/latest/specification/14_signals/14_01_introduction.html - Table 120) which results in something like this:

![image](https://github.com/esmini/esmini/assets/771979/3de59fd9-941c-4cd8-8e35-dbe31dd2f672)

Are there any plans to update to at least a subset of 1.8 any time soon?

Related: https://github.com/pyoscx/scenariogeneration/issues/221

**Comment by eknabevcc** [comment]
BTW: Could you share a scenario snippet including this stop line feature? And we'll have a look, at least into esmini part of it.

**Comment by johschmitz** [comment]
Here is an example:

[example.zip](https://github.com/user-attachments/files/16147987/example.zip)

You can run it with the `--use_signs_in_external_model` flag. Also I am thinking it should maybe be "signals" instead of "signs" (like `use_signals_from_external_model`), I hope it will also work for traffic lights even if it just says "signs"..

**Comment by eknabevcc** [comment]
That was extremely low hanging fruit, for a change! The signal length attribute turned out already to be supported by esmini.

See attached modified xodr, with length attribute added for the stop line.

[bdsc_export_xodr_mod.zip](https://github.com/user-attachments/files/16155750/bdsc_export_xodr_mod.zip)

![Image](https://github.com/esmini/esmini/assets/29146839/613a5509-84f6-40bb-9777-7d59be14c6de)

**Comment by johschmitz** [comment]
That's cool, I hope this will also be supported by the scenariogeneration lib soon. I tried to create a quick PR for it like you did but there it is a little bit more work because the OpenDRIVE .xsd files need to be added first I think..

**Comment by eknabevcc** [comment]
Yes, good that you provided the [PR](https://github.com/pyoscx/scenariogeneration/pull/244) to the scenariogeneration project. So that it can be considered and dependencies sorted out.
